pgcli/pgexecute.py

```diff
@@ -11,6 +12,9 @@
 psycopg2.extensions.register_type(psycopg2.extensions.UNICODE)
 psycopg2.extensions.register_type(psycopg2.extensions.UNICODEARRAY)
 
+# When running a query, make pressing CTRL+C raise a KeyboardInterrupt
+psycopg2.extensions.set_wait_callback(psycopg2.extras.wait_select)
```

Comment:
Yeah I noticed that in the article after I left the comment. Now I'm trying to find the source for wait_select() to check the difference between wait_select() and wait_select_inter().